alacritty_terminal/src/clipboard.rs

```diff
@@ -18,35 +18,49 @@ use std::ffi::c_void;
 use log::{debug, warn};
 
 use copypasta::nop_clipboard::NopClipboardContext;
-#[cfg(not(any(target_os = "macos", target_os = "windows")))]
+#[cfg(not(any(target_os = "macos", windows)))]
+#[cfg(feature = "wayland")]
```

Comment:
```suggestion
#[cfg(all(not(any(target_os = "macos", windows)), feature = "wayland")]
```

```suggestion
#[cfg(all(not(target_os = "macos"), not(windows)), feature = "wayland")]
```

Wouldn't one of these work?